server/src/metadata/standard-objects/companies/companies.metadata.ts

```diff
@@ -8,7 +8,7 @@ const companiesMetadata = {
   icon: 'IconBuildingSkyscraper',
   fields: [
     {
-      type: 'text',
+      type: 'TEXT',
```

Comment:
why are we changing this to caps? I can find occurrences of lower caps everywhere in the code, including in `FieldMetadataType`